torchgeo/datasets/l8biome.py

```diff
@@ -0,0 +1,237 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""L8Biome dataset."""
+
+import glob
+import os
+from typing import Any, Callable, Dict, List, Optional, cast
+import re
+
+from rasterio.crs import CRS
+
+from .geo import RasterDataset
+from .utils import BoundingBox, download_url, extract_archive
+from matplotlib.colors import ListedColormap
+from PIL import Image
+from rasterio.crs import CRS
+from torch import Tensor
+from torch.utils.data import Dataset
+import matplotlib.pyplot as plt
+import numpy as np
+
+class L8Biome(RasterDataset):
+    r"""L8 Biome datasets.
```

Comment:
```suggestion
    """L8 Biome datasets.
```
I don't think you're using any escapes anywhere